### PR TITLE
[FIX] product: adapt help tooltip

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Adapt to https://github.com/odoo/odoo/commit/65652c73ba5331f77911186543062dfddf50f077 and https://github.com/odoo/odoo/commit/728d9f83f6d140b57b91000f683298f1f5bd43c9.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Feyensv** [comment]
`detailed_type` field will be dropped in the coming days, no need to update its help.

**Comment by MiquelRForgeFlow** [comment]
So, should we close the PR or should just add the other help?

File: addons/stock/models/product.py [hunk 1]
```diff
@@ -659,7 +659,7 @@ class ProductTemplate(models.Model):
 
     is_storable = fields.Boolean(
         'Track Inventory', store=True, compute='compute_is_storable', readonly=False,
-        default=False, precompute=True)
+        default=False, precompute=True, help='A storable product is a product for which you manage stock.\n')
```
Comment (Feyensv) [inline addons/stock/models/product.py:662]: @dalaOdoo what would you expect as tooltip for the new `Track inventory` field ?

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**